lib/dynamoid/associations/many_association.rb

```diff
@@ -177,7 +177,8 @@ def destroy_all
       # @since 0.2.0
       def delete_all
         objs = target
-        source.update_attribute(source_attribute, nil)
+        source.write_attribute(source_attribute, nil)
+        source.save(skip_callbacks: true)
```

Comment:
Looks good. But it can be improved with `#update` or `update_fields` methods. A least in this case we don't need to add the `skip_callbacks` method. As far as I know, there is no similar option (for the `#save` method) in Rails.

Reply:
Thank you for your feedback!
I'll comeback with better changes.